.gitlab-ci.yml

```diff
@@ -93,9 +121,17 @@ Base:
       - "*.repo"
     when: always
 
+Base-nightly:
+  stage: test
+  extends: Base
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "schedule" && $RUNNER =~ /[\S]+\/rhel-8\..-x86_64/'
```

Comment:
Will fail when secondary arches are added.
Will fail when we add rhel-9 runners.

Why not use only "rhel" and possibly have a simpler regex?

Reply:
> Will fail when secondary arches are added.

True but I thought that for nightly testing we're only concerned with x86_64. No problem to add it though.

> Will fail when we add rhel-9 runners.
>
> Why not use only "rhel" and possibly have a simpler regex?

If I just add all RHEL runners I'm afraid that there will be a duplication of test runs. What this whole PR currently achieves is that it runs the tests on a 8.4 runner with 8.5 nightly repos. If I also include regular rhel-8 runner, that will just mean it will also run on 8.3 with 8.5 nightly repos, unless I'm missing something. So that's why I made the regexp to match only 8.X which should be the current nightly. 

About rhel-9 runners, sure I can add them now, didn't think about them ahead..